specification/cost-management/resource-manager/readme.go.md

```diff
@@ -49,3 +50,11 @@ Please also specify `--go-sdk-folder=<path to the root directory of your azure-s
 ``` yaml $(tag) == 'package-preview-2019-03' && $(go)
 output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2019-03-01/$(namespace)
 ```
+
+### Tag: package-2019-10 and go
+These settings apply only when `--tag=package-2019-10 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2019-10' && $(go)
+output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2019-10-01/$(namespace)
```

Comment:
Please change this to 
`$(go-sdk-folder)/services/preview/$(namespace)/mgmt/2019-10-01-preview/$(namespace)`